package.json

```diff
@@ -722,6 +722,26 @@
           "markdownDescription": "`#editor.lineNumbers#` is determined by the active Vim mode, absolute when in insert mode, relative otherwise.",
           "default": false
         },
+        "vim.targets.enable": {
+          "type": "boolean",
+          "markdownDescription": "Enable [targets.vim](https://github.com/wellle/targets.vim#quote-text-objects) plugin (not fully implmeneted yet).",
+          "default": false
+        },
+        "vim.targets.smartQuotes.enable": {
+          "type": "boolean",
+          "markdownDescription": "Enable smart quotes movements",
```

Comment:
This should end in a period and should probably include [this link](https://github.com/wellle/targets.vim#quote-text-objects)

Reply:
```suggestion
          "markdownDescription": "Enable the smart quotes movements from [targets.vim](https://github.com/wellle/targets.vim#quote-text-objects).",
```